### ISSUE TITLE
Release `0.1.0`

### ISSUE THREAD

**Issue by brillout** [hattipjs/hattip#67]
Are there blockers for publishing a non-experimental release?

Are there APIs that may change soon?

**Comment by aleclarson** [comment]
I agree. There's really no benefit of not bumping to v0.1 now. It makes it easier for projects to receive patches without manual intervention.
